W-14793637-chnge-endpoint-terminology (#174)
mulesoft/docs-hosting: cloudhub-2/modules/ROOT/pages/ch2-config-endpoints-paths.adoc

= Configuring Endpoints and Paths for Apps Deployed to a Private Space

[IMPORTANT]
CloudHub 2.0 supports customizing external endpoints only for apps that are deployed to private spaces, not shared spaces.

A private space routes requests from clients to apps deployed to the private space.

> When you deploy an app to a private space, both _external_ and _internal endpoints_ are created, but you can modify only the external endpoint.

Reviewer: ```suggestion
When you deploy an app to a private space, two types of _endpoints_ are created:

* *Endpoint*: A URL used to reach the app deployed to the public space. In this context, it is accessible externally and enables clients outside of the network to reach the app. You can delete this endpoint to prevent external traffic from reaching the app.
* *Internal endpoint*: A URL used to reach the app deployed to the public space, but it is accessible only within the private network, VPN, or transit gateway (TGW). This type of endpoint cannot be modified or deleted.

```

> An _endpoint_ is the URL used to reach the app deployed to the private space,

Reviewer: ```suggestion
```

> for example `\https://my-app.example.com/`. External endpoints enable clients outside of the network to reach the app. Internal endpoints are accessible only within the private network, a VPN, or transit gateway (TGW). To ensure an app is accessible only within the network, you can remove external endpoints.

Reviewer: ```suggestion
An example endpoint URL is `\https://my-app.example.com/`.
```
